tests/Request/InitTest.php

```diff
@@ -4,9 +4,21 @@
 
 use LearnositySdk\AbstractTestCase;
 use LearnositySdk\Exceptions\ValidationException;
+use LearnositySdk\Services\SignatureFactory;
 
 class InitTest extends AbstractTestCase
 {
+
+    /**
+     * @var \LearnositySdk\Services\Signatures\SignatureInterface|mixed
+     */
+    private $signatureGenerator;
+
+    public function setUp(): void
+    {
+        $this->signatureGenerator = SignatureFactory::getSignatureGenerator('02');
```

Comment:
I think we should keep the legacy Hash tests in place to prevent regression, and duplicate this file to add the HMAC tests.